pkg/manager/constants.go

```diff
@@ -29,6 +29,11 @@ const (
 	// manager option.
 	DefaultSyncPeriod = time.Minute * 10
 
+	// DefaultEnableKeepAlive is false by default
+	DefaultEnableKeepAlive = true
+
+	DefaultKeepAliveDuration = time.Duration(5)
```

Comment:
shall we be more specific, is this 5 min / 5 sec?

Reply:
Ack. DefaultEnableKeepalive changed to false.